src/main/kotlin/com/github/hanseter/json/editor/types/OneOfModel.kt

```diff
@@ -59,7 +61,7 @@ open class OneOfModel(
         if (!typeSetManually
             && (actualType == null || !isValid(actualType!!.model.schema.baseSchema, value))
         ) {
-            val possibleNewType = tryGuessActualType()
+            val possibleNewType = tryGuessActualTypeControl()
```

Comment:
Are we constructing this control eagerly now?

Reply:
I just changed the name of the function to make it more clear what happens. We create the actual control there when the binding changes